source/screaming-snake-case.d.ts

```diff
@@ -21,13 +11,13 @@ This can be useful when, for example, converting a camel-cased object property t
 import type {ScreamingSnakeCase} from 'type-fest';
 
 const someVariable: ScreamingSnakeCase<'fooBar'> = 'FOO_BAR';
+const someVariableNoSplitOnNumber: ScreamingSnakeCase<'p2pNetwork', {splitOnNumber: false}> = 'P2P_NETWORK';
+
 ```
 
 @category Change case
 @category Template literal
-*/
-export type ScreamingSnakeCase<Value> = Value extends string
-	? IsScreamingSnakeCase<Value> extends true
```

Comment:
I can bring this back, but I believe it shouldn't be necessary, especially that we don't use it for other things like snake case or kebab case